ajax/libs/socket.io/package.json

```diff
@@ -12,10 +12,9 @@
   "npmName": "socket.io-client",
```

Comment:
Because socket.io-client in npm directing to github repo is [this](https://github.com/socketio/socket.io-client).
Maybe we can modify the name of this lib to socket.io-client, I think that will be better.
Or it will confuse **socket.io** with **socket.io-client-**.